index.html

```diff
@@ -730,15 +730,8 @@ <h3>Unicode Normalization</h3>
           Unicode Normalization Form C.</p>
         <section id="canonical_compatibility">
           <h4>Canonical vs. Compatibility Equivalence</h4>
-          <p>Unicode defines two types of equivalence between characters: <em>canonical
-              equivalence</em> and <em>compatibility equivalence</em>.</p>
-          <p><dfn>Canonical equivalence</dfn> is a fundamental equivalency
-            between Unicode characters or sequences of Unicode characters that
-            represent the same abstract character. When correctly displayed,
-            these should always have the same visual appearance and behavior.
-            Generally speaking, two canonically equivalent Unicode texts should
-            be considered to be identical as text. Unicode defines a process called 
-            <em>canonical decomposition</em> that removes these primary distinctions between two texts.</p>
+          <p>Unicode defines two types of equivalence between characters: <em>canonical equivalence</em> and <em>compatibility equivalence</em>.</p>
+          <p><dfn>Canonical equivalence</dfn> is a fundamental equivalency between Unicode characters or sequences of Unicode characters that represent the same abstract character. Equivaent sequences, ideally should have the same visual appearance (although there are many factors that can cause them to appear somewhat differently) and they should be treated and processed equivalently. Generally speaking, two canonically equivalent Unicode texts should be considered to be identical as text. Unicode defines a process called <em>canonical decomposition</em> that removes these primary distinctions between two differently-encoded but canonically equivalent texts.</p>
```

Comment:
Would it be better to say "Unicode code points" here (not "Unicode characters")?